modules/CMakeLists.txt

```diff
@@ -30,6 +30,12 @@ if (";${TARGET_MODULES};" MATCHES ";CLOCK_CONFIG;")
   set(MODULES_FLAGS "${MODULES_FLAGS} -DMODULE_CLOCK_CONFIG")
 endif()
 
+if (";${TARGET_MODULES};" MATCHES ";ERROR;")
+  set(PROJ_SRCS ${PROJ_SRCS} ${MODULE_PATH}/mmc_error.c )
+  set(MODULES_FLAGS "${MODULES_FLAGS} -DMODULE_ERROR")
+endif()
+
+
```

Comment:
Remove this, the `mmc_error.c` file is already included in line 11:
https://github.com/lnls-dig/openMMC/blob/f067c06f05f00c6e52f919df6d4b6d6f01b975c7/modules/CMakeLists.txt#L11